composer.json

```diff
@@ -29,7 +29,7 @@
         "mimmi20/file-loader": "~2.0",
         "symfony/filesystem": "*",
         "symfony/finder": "~2.6",
-        "symfony/console": "~2.6",
+        "symfony/console": "^3.0",
```

Comment:
@asgrim if you allow both, you should also test the lowest/highest dependency.

Example:
https://github.com/ThaDafinser/UserAgentParser/blob/master/.travis.yml#L14-L15
https://travis-ci.org/ThaDafinser/UserAgentParser